pkg/cluster/cluster.go

```diff
@@ -205,7 +205,10 @@ func (c *Cluster) run(stopC <-chan struct{}, wg *sync.WaitGroup) {
 	defer func() {
 		if needDeleteCluster {
 			c.logger.Infof("deleting cluster")
-			c.delete()
+			err := c.delete()
+			if err != nil {
+				logrus.Errorf("fail to delete cluster: %v", err)
```

Comment:
use c.logger. also the logger is already prefixed by cluster name. do not repeat it.